SimulationPool.java

```diff
@@ -41,7 +49,12 @@ public SimulationPool(int games, double [][] weightSets) {
        
        	//For each generation, we run a 1000 games. For each game, we assign a thread.
         for(int i=0; i< games; i++){
-            Simulation game = new Simulation(weightSets[i]);
+            Simulation game;
+            if (isGA) {
+                game = new Simulation(weightSets[i]);
+            } else {
+                game = new Simulation(weightSets[0]);
```

Comment:
I tried running PSO, and i realised that SimulationPool is just running a single sets of weights 500 times. Not the intended function I think?

![image](https://user-images.githubusercontent.com/27397021/37752089-066fb94c-2dd1-11e8-80e0-a76ff82f309a.png)

I will make another branch which you can pull to simply study what is happening now

Reply:
#10